src/main/java/edu/ie3/datamodel/utils/ExceptionUtils.java

```diff
@@ -24,4 +27,14 @@ public static String getMessages(List<? extends Exception> exceptions) {
         .reduce("", (a, b) -> a + "\n " + b)
         .replaceFirst("\n ", "");
   }
+
+  /**
+   * Combines multiple {@link UniqueEntity} into a string.
+   *
+   * @param entities to be combined
+   * @return a string
+   */
+  public static String combine(Collection<? extends UniqueEntity> entities) {
+    return entities.stream().map(UniqueEntity::toString).collect(Collectors.joining("\n - "));
```

Comment:
The first item is placed in the same line, all following items are placed in consecutive lines. Maybe you can find a better solution here.